docsrc/manual/scripting-capabilities.rst

```diff
@@ -1010,6 +1010,79 @@ All overlay functions are callable via the global ``overlay`` object.
 
    :param number opacity: the opacity
 
+.. js:function:: overlay.getHorizontalScale(layer = 0)
+
+   Gets the horizontal scale of the specified overlay layer. ``1.0`` is normal size.
+
+   :param number layer: the layer id. Defaults to ``0``
+   :returns number: the scale
+
+.. js:function:: overlay.getVerticalScale(layer = 0)
+
+   Gets the vertical scale of the specified overlay layer. ``1.0`` is normal size.
+
+   :param number layer: the layer id. Defaults to ``0``
+   :returns number: the scale
+
+.. js:function:: overlay.setHorizontalScale(scale, layer)
+
+   Sets the horizontal scale of the specified overlay layer. ``1.0`` is normal size.
+
+   :param number scale: the scale to set
+   :param number layer: the layer id
+
+.. js:function:: overlay.setHorizontalScale(scale)
+
+   This is an overloaded function. Sets the horizontal scale of all active overlay layers. Layers that have not been used yet will not have their scale changed. ``1.0`` is normal size.
+
+   :param number scale: the scale to set
+
+.. js:function:: overlay.setVerticalScale(scale, layer)
+
+   Sets the vertical scale of the specified overlay layer. ``1.0`` is normal size.
+
+   :param number scale: the scale to set
+   :param number layer: the layer id
+
+.. js:function:: overlay.setVerticalScale(scale)
+
+   This is an overloaded function. Sets the vertical scale of all active overlay layers. Layers that have not been used yet will not have their scale changed. ``1.0`` is normal size.
+
+   :param number scale: the scale to set
+
+.. js:function:: overlay.getAngle(layer = 0)
```

Comment:
These `angle`-named functions should use the word `rotation` instead.  I.e. `getRotation`, `setRotation`.

Reply:
Yeah I had them that way originally but I renamed them because I thought it was less accurate. It probably makes more sense for them to use a cohesive name, I'll revert it